cafebabel/templates/articles/macros.html

```diff
@@ -15,3 +15,21 @@
     Reading time: {{ article.body|reading_time }} min
   </p>
 {%- endmacro %}
+
+{%- macro tags_fields(article=None) -%}
+  <div class=tags>
+    <label for=tag-1>Tags</label>
+    <ul class=tags-list>
+      {% if article and article.tags %}
+        {% for tag in article.tags -%}
+          <li>
+            {{ tag.name }}
+            <input name=tag-{{ loop.index }} list=tags value="{{ tag.name }}" type=hidden>
+          </li>
+        {%- endfor %}
+      {% endif %}
+    </ul>
+    <div class=tag-container><input name=tag-new autocomplete=off list=tags-suggestions><button class=add>+</button></div>
+    <ul id=tags-suggestions class=inactive></ul>
```

Comment:
@cafebabel-webmaster changing that `datalist` element to a `ul` probably broke the native autocomplete from the browser, not sure that is something we want? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist

Reply:
I couldn't style the datalist elements. That's why I switched for `ul` and `li`. we can talk about it later